pyaedt/edb_grpc/core/edb_data/hfss_simulation_setup_data.py

```diff
@@ -0,0 +1,2146 @@
+        low_freq_adapt_data.MaxPasses = max_num_passes
+        low_freq_adapt_data.AdaptiveFrequency = self._parent._edb.edb_value(low_frequency).ToString()
+        high_freq_adapt_data = self._parent._edb.simsetupdata.AdaptiveFrequencyData()
+        high_freq_adapt_data.MaxDelta = self._parent._edb.edb_value(max_delta_s).ToString()
+        high_freq_adapt_data.MaxPasses = max_num_passes
+        high_freq_adapt_data.AdaptiveFrequency = self._parent._edb.edb_value(high_frequency).ToString()
+        self.adaptive_settings.AdaptiveFrequencyDataList.Add(low_freq_adapt_data)
+        self.adaptive_settings.AdaptiveFrequencyDataList.Add(high_freq_adapt_data)
+        return self._parent._update_setup()
+
+
+class DefeatureSettings(object):
+    """Manages EDB methods for defeature settings."""
+
+    def __init__(self, parent):
+        self._parent = parent
+
+    @property
+    def _defeature_settings(self):
+        return self._parent._edb_sim_setup_info.SimulationSettings.DefeatureSettings
+
+    @property
+    def defeature_abs_length(self):
+        """Absolute length for polygon defeaturing.
+
+        Returns
+        -------
+        str
+        """
+        return self._defeature_settings.DefeatureAbsLength
+
+    @defeature_abs_length.setter
+    def defeature_abs_length(self, value):
+        self._defeature_settings.DefeatureAbsLength = value
+        self._parent._update_setup()
+
+    @property
+    def defeature_ratio(self):
+        """Defeature ratio.
+
+        Returns
+        -------
+        float
+        """
+        return self._defeature_settings.DefeatureRatio
+
+    @defeature_ratio.setter
+    def defeature_ratio(self, value):
+        self._defeature_settings.DefeatureRatio = value
+        self._parent._update_setup()
+
+    @property
+    def healing_option(self):
+        """Whether to turn on healing of mis-aligned points and edges.
+        Options are:
+        0- Turn off.
+        1- Turn on.
+
+        Returns
+        -------
+        int
+        """
+        return self._defeature_settings.HealingOption
+
+    @healing_option.setter
+    def healing_option(self, value):
+        self._defeature_settings.HealingOption = value
+        self._parent._update_setup()
+
+    @property
+    def model_type(self):
+        """Model type.
+        Options:
+        0- General.
+        1- IC.
+
+        Returns
+        -------
+        int
+        """
+        return self._defeature_settings.ModelType
+
+    @model_type.setter
+    def model_type(self, value):
+        """Model type (General 0 or IC 1)."""
+        self._defeature_settings.ModelType = value
+        self._parent._update_setup()
+
+    @property
+    def remove_floating_geometry(self):
+        """Whether to remove floating geometries.
+
+        Returns
+        -------
+        bool
+            ``True`` if floating geometry removal is used, ``False`` otherwise.
```

Comment:
```suggestion
            ``True`` if floating geometry removal is used, ``False`` otherwise.
```
```suggestion
            ``True`` if floating geometry is removed, ``False`` otherwise.
```